index.mako

```diff
@@ -467,8 +467,8 @@ file that tells pre-commit:
     =c= (optional: default `[]`) list of additional parameters to pass to the hook.
 =r=
     =c= [`stages`](_#hooks-stages)
-    =c= (optional) confines the hook to the `commit`, `merge-commit`, `push`,
-        `prepare-commit-msg`, `commit-msg`, `post-checkout`, `post-commit`, or
+    =c= (optional: default `[commit, push]`) confines the hook to the `commit`, `merge-commit`,
```

Comment:
no no, I mean copy the text from there, the default is all stages just like `default_stages`: https://pre-commit.com/#top_level-default_stages

Reply:
Oh sure, sorry about that.